[2025-09-03T14:54:45.168+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T14:54:45.194+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T01:05:00+00:00 [queued]>
[2025-09-03T14:54:45.203+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T01:05:00+00:00 [queued]>
[2025-09-03T14:54:45.203+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T14:54:45.348+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-03 01:05:00+00:00
[2025-09-03T14:54:45.360+0000] {standard_task_runner.py:63} INFO - Started process 2766033 to run task
[2025-09-03T14:54:45.365+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-03T01:05:00+00:00', '--job-id', '74943', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmp21pz5xcw']
[2025-09-03T14:54:45.368+0000] {standard_task_runner.py:91} INFO - Job 74943: Subtask save_foreign_1
[2025-09-03T14:54:45.424+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T01:05:00+00:00 [running]> on host 1877907bc043
[2025-09-03T14:54:45.525+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T01:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T01:05:00+00:00'
[2025-09-03T14:54:45.526+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T14:54:45.541+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:54:46.045+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-03T14:54:46.047+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:54:46.673+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-03T14:54:46.674+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:54:47.200+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-03T14:54:47.202+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:54:47.787+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-03T14:54:47.788+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T14:54:47.789+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T14:54:47.803+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-03T01:05:00+00:00, execution_date=20250903T010500, start_date=20250903T145445, end_date=20250903T145447
[2025-09-03T14:54:47.985+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T14:54:48.024+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-03T14:54:48.025+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
